config/build.conf.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
just curious, why is this needed here?

Reply:
@mcasimir It’s not needed for anything in particular, it’s just good practice to always write JS code in strict mode (TS should already always be strict mode for us)